docsv2/bug-reports.md

```diff
@@ -0,0 +1,31 @@
+# Reporting Issues
+
+> [!Important]
+> Change the link in `.github/ISSUE_TEMPLATE/bug-report.yaml` once you change/move the filename.
+
+If you require support with a bug, you can help us help you.
+Please understand that Tasks is mostly maintained by a single person in her free time.
+
+It's crucial that we receive a **text representation** of your files in order to be able to investigate issues.
+Trying to reproduce an issue based on a description or screenshot is time-intensive and error-prone.
+Especially since there exists a plethora of themes and CSS-snippets which change how your tasks and checkboxes look.
+
+Please understand that we don't have the time to chase after details when a text representation of the vault already includes everything we need.
+
+Here is how you can send us a useful text representation:
+
+1. Find all the files that are required to reproduce the issue
+2. Find Tasks' settings file in your vault under `.obsidian/plugins/obsidian-tasks-plugin/data.json`
+    - The `.obsidian` directory is in the `/` root of your vault.
+    - Note that, on most operating systems, the directory `.obsidian` is hidden by default. Please check your operating system's help to find out how to make it show up in your file browser.
```

Comment:
This is really interesting, and really well written. Thank you...

I hadn't thought of asking for a vault.

On the one hand, it will be easier for me to open and reproduce the problem.

On the other hand, would I need to inspect the contents for malicious scripts before opening the vault?

I think we should try this and see how it goes. We can always fall back on 'here is how you format markdown on GitHub in a later version, if the zip approach becomes too onerous.